src/test/java/com/example/finance/service/UserAccountServiceTest.java

```diff
@@ -0,0 +1,110 @@
+package com.example.finance.service;
+
+import com.example.finance.exception.model.BackendException;
+import com.example.finance.mapper.UserAccountMapper;
+import com.example.finance.model.dto.UserAccountDto;
+import com.example.finance.model.entity.UserAccountEntity;
+import com.example.finance.model.enums.UserRole;
+import com.example.finance.repository.UserAccountRepository;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class UserAccountServiceTest {
+
+    @Mock
+    private UserAccountRepository userAccountRepository;
+
+    @Mock
+    private UserAccountMapper userAccountMapper;
+
+    @Mock
+    private PasswordEncoder passwordEncoder;
+
+    @InjectMocks
+    private UserAccountService userAccountService;
+
+    @Test
+    void getByLoginAndPassword_Success() {
```

Comment:
please use given naming convention to test method names -> methodName_testedCase_expectedResult

Reply:
please use given naming convention